Automattic/newspack-plugin
# Setup Wizard Graphic

### All Submissions:






### Changes proposed in this Pull Request:

Adds graphic to the Welcome screen of Set Up Wizard.

<img width="1399" alt="Screen Shot 2019-07-02 at 8 05 36 AM" src="https://user-images.githubusercontent.com/1477002/60511569-481be980-9ca0-11e9-99e3-5e693ad73b41.png">

Closes #150 .

### How to test the changes in this Pull Request:

1. `npm ci && npm run clean && npm run build:webpack`
2. Click Set Up under Newspack menu.
3. Observe graphic on the welcome screen.

### Other information:
